lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/ImruJobCancelledTests.cs

```diff
@@ -0,0 +1,254 @@
+                .BindNamedParameter(typeof(SerializedMapTaskStateConfiguration),
+                    _configurationSerializer.ToString(jobDefinition.MapTaskStateConfiguration))
+                .BindNamedParameter(typeof(SerializedMapConfiguration),
+                    _configurationSerializer.ToString(jobDefinition.MapFunctionConfiguration))
+                .BindNamedParameter(typeof(SerializedUpdateConfiguration),
+                    _configurationSerializer.ToString(jobDefinition.UpdateFunctionConfiguration))
+                .BindNamedParameter(typeof(SerializedMapInputCodecConfiguration),
+                    _configurationSerializer.ToString(jobDefinition.MapInputCodecConfiguration))
+                .BindNamedParameter(typeof(SerializedMapInputPipelineDataConverterConfiguration),
+                    _configurationSerializer.ToString(jobDefinition.MapInputPipelineDataConverterConfiguration))
+                .BindNamedParameter(typeof(SerializedUpdateFunctionCodecsConfiguration),
+                    _configurationSerializer.ToString(jobDefinition.UpdateFunctionCodecsConfiguration))
+                .BindNamedParameter(typeof(SerializedMapOutputPipelineDataConverterConfiguration),
+                    _configurationSerializer.ToString(jobDefinition.MapOutputPipelineDataConverterConfiguration))
+                .BindNamedParameter(typeof(SerializedReduceConfiguration),
+                    _configurationSerializer.ToString(jobDefinition.ReduceFunctionConfiguration))
+                .BindNamedParameter(typeof(SerializedResultHandlerConfiguration),
+                    _configurationSerializer.ToString(jobDefinition.ResultHandlerConfiguration))
+                .BindNamedParameter(typeof(MemoryPerMapper),
+                    jobDefinition.MapperMemory.ToString(CultureInfo.InvariantCulture))
+                .BindNamedParameter(typeof(MemoryForUpdateTask),
+                    jobDefinition.UpdateTaskMemory.ToString(CultureInfo.InvariantCulture))
+                .BindNamedParameter(typeof(CoresPerMapper),
+                    jobDefinition.MapTaskCores.ToString(CultureInfo.InvariantCulture))
+                .BindNamedParameter(typeof(CoresForUpdateTask),
+                    jobDefinition.UpdateTaskCores.ToString(CultureInfo.InvariantCulture))
+                .BindNamedParameter(typeof(MaxRetryNumberInRecovery),
+                    jobDefinition.MaxRetryNumberInRecovery.ToString(CultureInfo.InvariantCulture))
+                .BindNamedParameter(typeof(InvokeGC),
+                    jobDefinition.InvokeGarbageCollectorAfterIteration.ToString(CultureInfo.InvariantCulture))
+                .Build();
+                
+            return driverConfig;
+        }
+
+        /// <summary>
+        /// Simple Type to help with Tang injection when constructing IMRUDriver.
+        /// Cares minimum implmentation to satisfy new driver instance for test scenarios
```

Comment:
fixed

---
In reply to: [86019354](https://github.com/apache/reef/pull/1161#discussion_r86019354) [](ancestors = 86019354)